enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -131,7 +132,9 @@ func runSociStore(ctx context.Context) {
 
 func NewProvider(env environment.Env, imageCacheAuthenticator *container.ImageCacheAuthenticator, buildRoot string) (*Provider, error) {
 	if *imageStreamingEnabled || len(*streamableImages) > 0 {
-		go runSociStore(env.GetServerContext())
+		if *runSociSnapshotter {
```

Comment:
Why is it a problem for a local development? Instead of running soci snapshotter manually could you not let the executor start it instead? Would that solve your problem?

Reply:
I meant for developing the snapshotter locally and running a custom version without putting it into /usr/bin.

Reply:
Would it solve your problem if you made the path to soci-store a flag so you can set it locally to whatever location you want?